src/executePrettier/buildPrettierOptions.test.js

```diff
@@ -109,40 +107,46 @@ it('uses vue as the parser if current scope is listed as a Vue SFC scope in sett
   expect(actual).toEqual({ parser: 'vue' });
 });
 
-it('does not use editorconfig options if that setting is not enabled', () => {
+it('passes editorconfig option to prettier', () => {
+  let editorconfig = false;
+
   const editor = buildMockEditor();
+  const mockPrettierInstance = { resolveConfig: { sync: jest.fn() } };
+  getPrettierInstance.mockImplementation(() => mockPrettierInstance);
+  shouldUseEditorConfig.mockImplementation(() => editorconfig);
+  getCurrentFilePath.mockImplementation(() => 'parent-dir/foo.js');
   getPrettierOptions.mockImplementation(() => ({}));
-  shouldUseEditorConfig.mockImplementation(() => false);
-
   buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[0][1]).toEqual({ editorconfig });
 
-  expect(buildEditorConfigOptions).not.toHaveBeenCalled();
+  editorconfig = true;
+  buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[1][1]).toEqual({ editorconfig });
 });
 
-it('overrides values with editorconfig values if editor config is enabled', () => {
+it('only uses project prettier config when found', () => {
   const editor = buildMockEditor();
   const fakePrettierOptions = { tabWidth: 2, printWidth: 80, useTabs: false };
```

Comment:
let's strike this line since it shouldn't be used

Reply:
whoops - thanks